src/model/game/GameManager.ts

```diff
@@ -492,6 +492,24 @@ export default class GameManager {
             "https://thunderstore.io/c/touhou-lost-branch-of-legend/api/v1/package/", "https://raw.githubusercontent.com/ebkr/r2modmanPlus/master/modExclusions.md",
             [new StorePlatformMetadata(StorePlatform.STEAM, "1140150")], "TouhouLostBranchOfLegend.jpg",
             GameSelectionDisplayMode.VISIBLE, GameInstanceType.GAME, PackageLoader.BEPINEX, ["lbol"]),
+
+        new Game("Sunkenland", "Sunkenland", "Sunkenland",
+            "Sunkenland", ["Sunkenland.exe"], "Sunkenland_Data",
+            "https://thunderstore.io/c/sunkenland/api/v1/package/", "https://raw.githubusercontent.com/ebkr/r2modmanPlus/master/modExclusions.md",
+            [new StorePlatformMetadata(StorePlatform.STEAM, "2080690")], "Sunkenland.jpg",
+            GameSelectionDisplayMode.VISIBLE, GameInstanceType.GAME, PackageLoader.BEPINEX, ["sunk"]),
+
+        new Game("Atomicrops", "Atomicrops", "Atomicrops",
+            "Atomicrops", ["Atomicrops.exe"], "Atomicrops_Data",
+            "https://thunderstore.io/c/atomicrops/api/v1/package/", "https://raw.githubusercontent.com/ebkr/r2modmanPlus/master/modExclusions.md",
+            [new StorePlatformMetadata(StorePlatform.STEAM, "757320")], "Atomicrops.jpg",
+            GameSelectionDisplayMode.VISIBLE, GameInstanceType.GAME, PackageLoader.BEPINEX, ["atom"]),
```

Comment:
`["atom"]` is redundant since it's already included in the name of the game & thus discoverable by search. You can leave this as an empty array (applies for the others too)

Reply:
@Oksamies now it's an empty string, which is potentially even worse since in some matching schemes it would match _every_ string. It should be an empty array